### DISCUSSION TITLE
How can i use it in local

### DISCUSSION THREAD

**Discussion by yangliguo7** [Skyvern-AI/skyvern#1358] [Q&A]
<img width="1225" alt="image" src="https://github.com/user-attachments/assets/0d81369b-9b93-470d-96fb-569745289909">

how can i get LLM provider key ? thanks

**Comment by suchintan** [comment]
I think you'll need to get an API key from one of the LLMs listed here: https://github.com/Skyvern-AI/Skyvern?tab=readme-ov-file#environment-variables
